bot/prisma/migrations/20230406213618_initial_migration/migration.sql

```diff
@@ -0,0 +1,58 @@
+-- CreateTable
```

Comment:
I'm a little skeptical of a transition to postres as this stage.  It's very quick and easy to run `redis`, and it translate easily if we decided to migrate to something like kvstore and cloudflare workers.  Introducing schemas and a full relational database introduces a fair bit of complexity, and I want to be 100% sure it's needed.

Reply:
I originally implemented postgres (for performance reasons) into airhorn before the slash commands release, but was switched to redis upon request for simplicity.

The reason for switching to postgres now is that the bot does not scale well in terms of performance due to having to scan the keys to get total counts (try running `/airhornmeta stats` on the current production bot, it takes over 15 seconds). I have been running my own bot [Bread](https://github.com/thatskyz/bread) which uses postgres in the same way as I've implemented it here, and I have tested it in a simulated environment with over 66 million usages and it responds more than 80% faster. (I can talk to you more about this, shoot me a message.)

I understand Discords' push to have less bots using the gateway (you mentioned Cloudflare Workers here), but for airhorn it is not doable due to the following:
- In order to connect to the voice servers, the bot gets updates over the websocket
- To get the current voice state of the user who ran the command, I need to use the gateway events